app/src/androidTest/java/in/ac/bits_pilani/goa/ard/activities/AboutMACActivityTest.java

```diff
@@ -0,0 +1,103 @@
+package in.ac.bits_pilani.goa.ard.activities;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.support.v7.widget.Toolbar;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+import in.ac.bits_pilani.goa.ard.R;
+import in.ac.bits_pilani.goa.ard.utils.AHC;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by Aayush on 1/6/17.
+ * Test for AboutMAC activity.
+ */
+
+@RunWith(AndroidJUnit4.class)
+public class AboutMACActivityTest {
+
+    @Mock
```

Comment:
you can remove `@Mock` here it is not being mocked. We are getting the context from registry